CONTRIBUTING.md

```diff
@@ -1,95 +1,151 @@
+4. Issue closure: Follow the [guidance](https://docs.github.com/en/issues/tracking-your-work-with-issues/closing-an-issue)
+   - Close as completed once PRs are merged.
+   - Close as completed once the author accepts the `wont fix` decision.
+   - Close as not planned if a bug cannot be reproduced with the author's agreement. Create a new issue if this bug is reproduced in the future.
+   - Close as not planned by transforming the issue to GitHub discussion with the author's agreement, if an issue contains open-ended conversations that don’t necessarily have a specific goal or outcome in mind.
 
- - please read other user comments first, and confirm this is the same issue: a given error condition might be indicative of different problems - you may also find a workaround in the comments
- - please refrain from adding "same thing here" or "+1" comments
- - you don't need to comment on an issue to get notified of updates: just hit the "subscribe" button
- - comment if you have some new, technical and relevant information to add to the case
+### Stale issues or PRs
 
-### If you have not found an existing issue that describes your problem:
+A stale issue is one that remains inactive or without updates for a period of 60 days. A stale pull request (PR) is one that remains inactive or without updates for a period of 45 days. When an issue or PR becomes stale, it is labelled as `stale`. Normally maintainers will comment on stale issue or PR to prompt participants to take action. If there is no activity for additional 30 days, this issue or PR will be closed. If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart.
 
- 1. create a new issue, with a succinct title that describes your issue:
-   - bad title: "It doesn't work with my docker"
-   - good title: "Publish fail: 400 error with E_INVALID_DIGEST"
- 2. copy the output of:
-   - `notary version` or `docker version`
- 3. Run `notary` or `docker` with the `-D` option for debug output, and please include a copy of the command and the output.
- 4. If relevant, copy your `notaryserver` and `notarysigner` logs that show the error (this is likely the output from running `docker-compose up`)
+## Milestones
 
-## Contributing a patch for a known bug, or a small correction
+The Notary Project uses [milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) to track progress on groups of issues or pull requests in a repository. The following table defines the milestones that used by the Notary Project.
 
-You should follow the basic GitHub workflow:
+| Milestone           | Description                                                                                                                       |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
"discuss" and "future" are a bit vague. Wouldn't labels like "in discussion" and "backlog" be better?